src/Microsoft.AspNetCore.Server.HttpSys/Microsoft.AspNetCore.Server.HttpSys.csproj

```diff
@@ -10,6 +10,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\..\shared\Microsoft.AspNetCore.HttpSys.Sources\HeaderEncoding.cs" Link="RequestProcessing\HeaderEncoding.cs" />
```

Comment:
wildcard include? https://github.com/aspnet/StaticFiles/blob/1e0fe87c204b0199b97bc1d74f095e9746e9e674/samples/StaticFileSample/StaticFileSample.csproj#L8

Reply:
I may be doing this incorrectly, but I tried to link this in the project and it doesn't work :(.

Reply:
I assume it's the Link aspect that's giving you trouble. You should be able to remove that. If you want a certain folder structure then put that structure into the Sources package.